securedrop/utils.py

```diff
@@ -7,3 +11,23 @@ def validate_and_normalize_email(email):
         return valid.email
     except EmailNotValidError as e:
         print(str(e))
+
+
+def sha256_file(path: str):
+    if not os.path.exists(path):
+        return None
+
+    with open(path, "rb") as file:
+        hasher = SHA256.new()
```

Comment:
Just curious as to why you are using the := operation?

Reply:
Saves a line